packages/elements/src/icons/misc/Icons-misc.stories.tsx

```diff
@@ -1,22 +1,23 @@
 import * as React from "react";
-import { stenaArrowRight, stenaSearch } from "./Icons";
+
+import { stenaCombined } from "./Icons-misc";
 import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
 import { cssColor } from "@stenajs-webui/theme";
 import { Box, Row, Text } from "@stenajs-webui/core";
-import { Icon } from "../components/ui/icon/Icon";
+import { Icon } from "../../components/ui/icon/Icon";
+import _ from "lodash";
```

Comment:
Don't import all of lodash.

```suggestion
import { stuff } from "lodash";
```